docs/environment-variables.md

```diff
@@ -22,35 +20,56 @@ The [.env.example](../.env.example) file contains the essential environment vari
-
-## Azure Hosting (DevOps pipeline)
-
-These steps describe the process for making environment variables available to the the Azure DevOps pipeline.
-
-1. Declare the desired variable in the appropriate "variable group" in the Library section of the Azure DevOps site (https://dfe-ssp.visualstudio.com/Become-A-Teacher/_library?itemType=VariableGroups). All variable groups related to apply are suffixed as such and there is a variable group per deployment environment.
-2. Add your variable with the same name in all the 5(DevOps, QA, Staging, Sandbox, Production) variable groups.
-3. In [deploy.yml](../azure/pipelines/templates/deploy.yml#L191) make the following additions:
-   1. In the Azure Resource Group deployment task *overrideParameters* section add your variable in the format `-varName: "$(varName)"`
-4. In [azure/template.json](../azure/template.json) make the following additions:
-   1. Duplicate this block of code for your new variable at the end of the *parameters* section.
-      ```json
-      "varName": {
-        "type": "string",
-        "metadata": {
-          "description": "Describe your variable here."
-        }
-      }
-      ```
-   2. Around [line 500](../azure/template.json#L505) duplicate this block of code in the *appEnvironmentVariables* parameter of the `app-service-and-containers` resource and configure it to match your new environment varaible.
-   If the environment variable in question is a secret, change `value` to `secureValue`.
-      ```json
-      {
-        "name": "ENV_VAR_NAME",
-        "value": "[parameters('varName')]"
-      }
-      ```
-   The values asigned to `appEnvironmentVariables` will be available as environment variables in the web app, clock and worker container instances.
+For docker compose to make the necessary environment variables available in the container at run time they must be declared in the relevant docker-compose file.
+
+* [docker-compose.yml](../docker-compose.yml) - Variables that are required for local dev should be defined the *environment* section in this file. This will in general only be for the database.
+
+## Deploy Pipeline
+
+Runtime environment variables and app secrets are stored in Azure KeyVault and the github actions workflow reads this secrets and configures them as environment variables during the deployment. All required app secrets and environment variables are stored as one YAML file inside Azure KeyVault and each key in the file is set as individual environment variable as part of the deployment.
+Run the below commands from the root of the repository to view the current values stored in Key Vault
```

Comment:
could we also add that for those command to work that the Azure CLI needs to be installed?